### ISSUE TITLE
AppVeyor builds failing due to expired certificate

### ISSUE THREAD

**Issue by jwhui** [openthread/openthread#2325]
Example log snippet:
```
OpenThread.vcxproj -> ..\..\build\bin\x64\Release\app\OpenThread.pdb (Full PDB)
C:\Program Files (x86)\MSBuild\Microsoft\VisualStudio\v14.0\AppxPackage\Microsoft.AppXPackage.Targets(1930,5): warning APPX0108: The certificate specified has expired. For more information about renewing certificates, see http://go.microsoft.com/fwlink/?LinkID=241478. [C:\projects\openthread\etc\visual-studio\OpenThread.vcxproj]
  OpenThread -> C:\projects\openthread\build\bin\AppPackages\OpenThread_1.0.12.0_x64_Test\OpenThread_1.0.12.0_x64.appx
C:\Program Files (x86)\MSBuild\Microsoft\VisualStudio\v14.0\AppxPackage\Microsoft.AppXPackage.Targets(2557,5): error APPX0108: The certificate specified has expired. For more information about renewing certificates, see http://go.microsoft.com/fwlink/?LinkID=241478. [C:\projects\openthread\etc\visual-studio\OpenThread.vcxproj]
```

**Comment by jwhui** [comment]
Submitted #2329 as a temporary workaround.

**Comment by jwhui** [comment]
Merged #2329 as a temporary workaround.  Lowering the priority of this issue.
